plugins/outputs/redistimeseries/redistimeseries_test.go

```diff
@@ -133,15 +133,15 @@ func TestCases(t *testing.T) {
 			require.NoError(t, err)
 
 			// // Check the metric nevertheless as we might get some metrics despite errors.
-			actual := getAllRecords(address)
+			actual := getAllRecords(t, address)
 			require.ElementsMatch(t, expected, actual)
 		})
 	}
 }
 
-func getAllRecords(address string) []string {
+func getAllRecords(t *testing.T, address string) []string {
```

Comment:
I think we should pass the context here instead of `t`.

Reply:
Context passed.